src/main/scala/com/fulcrumgenomics/umi/ConsensusCaller.scala

```diff
@@ -141,6 +159,22 @@ class ConsensusCaller(errorRatePreLabeling:  PhredScore,
       val base = DnaBasesUpperCase(maxLlIndex)
       (base, q)
     }
+
+    /**
+      * Returns the calculated likelihoods of the four bases given the data and the model. The likelihoods
+      * returned factor in both the base observations and the probability of error prior to applying the
+      * labels.
+      */
+    def logLikelihoods: Array[LogProbability] = {
+      // get the sum of the likelihoods
+      // pick the base with the maximum posterior
```

Comment:
one line comment `get the sum of the likelihoods and pick the base with the maximum posterior`